js/clipboard.js

```diff
@@ -182,39 +182,39 @@
         return;
       }
 
+      let $form = $("<form />", {
+        "id": "sendForm",
+        "action": $sendButton.data("url"),
+        "method": $sendButton.data("method"),
+      });
+
       // Generate clipboard send data
       var data = {base_url: $sendButton.data('site-base-url')};
```

Comment:
I thing you could get rid of this `data` variable and add the values to the form inputs directly.